experimental/relativetime/src/relativetime.rs

```diff
@@ -0,0 +1,256 @@
+                })?
+                .take_payload()?;
+            let rt = rt.cast();
+            Ok(RelativeTimeFormatter {
+                plural_rules,
+                options,
+                rt,
+                fixed_decimal_format,
+            })
+        }
+    };
+}
+
+impl RelativeTimeFormatter {
+    constructor!(
+        try_new_long_second_unstable,
+        LongSecondRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_long_minute_unstable,
+        LongMinuteRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_long_hour_unstable,
+        LongHourRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_long_day_unstable,
+        LongDayRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_long_week_unstable,
+        LongWeekRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_long_month_unstable,
+        LongMonthRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_long_quarter_unstable,
+        LongQuarterRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_long_year_unstable,
+        LongYearRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_short_second_unstable,
+        ShortSecondRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_short_minute_unstable,
+        ShortMinuteRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_short_hour_unstable,
+        ShortHourRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_short_day_unstable,
+        ShortDayRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_short_week_unstable,
+        ShortWeekRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_short_month_unstable,
+        ShortMonthRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_short_quarter_unstable,
+        ShortQuarterRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_short_year_unstable,
+        ShortYearRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_narrow_second_unstable,
+        NarrowSecondRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_narrow_minute_unstable,
+        NarrowMinuteRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_narrow_hour_unstable,
+        NarrowHourRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_narrow_day_unstable,
+        NarrowDayRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_narrow_week_unstable,
+        NarrowWeekRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_narrow_month_unstable,
+        NarrowMonthRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
+        try_new_narrow_quarter_unstable,
+        NarrowQuarterRelativeTimeFormatDataV1Marker
+    );
+    constructor!(
```

Comment:
Nit: Make sure these are covered by tests. I suggest adding a unit test block in this file where you call every function for one of the test locales (maybe just English) just to make sure they work.

Reply:
Added tests that cover all constructors.